kobuki/b166c93.bug

```diff
@@ -48,7 +48,12 @@ bug:
 
   reproducibility: always
 
-  trace:
+  reproduction-images:
+    buggy: 57b89147438e41c46c0a6890ac7082a70c0056c2
+    fixed: b166c93913e293c55a3faa7b298c5f52f6def964
+    rosdistro: d0e0b0bfa624b65e40c81a6b308b84cf54f73ba7
```

Comment:
@wasowski: should we include repo links/identifiers here?

So: `ros/rosdistro@d0e0b0bfa624b65e40c81a6b308b84cf54f73ba7`?

Reply:
@wasowski?

Reply:
This is slightly redundant, as this info is static, but I agree that this would improve readability. I was quite confused by this example, and head to reread the doc.  So yes, I am in favour of adding the names.

Reply:
But will you remember to update `doc/taxonomy-description.tex` to reflect this, @gavanderhoorn ?